core/src/main/java/io/snyk/plugins/artifactory/scanner/ScannerModule.java

```diff
@@ -113,35 +126,35 @@ protected void validateVulnerabilityIssues(TestResult testResult, RepoPath repoP
     final String vulnerabilitiesForceDownload = repositories.getProperty(repoPath, vulnerabilitiesForceDownloadProperty);
     final boolean forceDownload = "true".equalsIgnoreCase(vulnerabilitiesForceDownload);
     if (forceDownload) {
-      LOG.info("Property '{}' is true, so we allow to download artifact: {}", vulnerabilitiesForceDownloadProperty, repoPath);
+      LOG.debug("Allowing download. Artifact Property \"{}\" is \"true\". {}", vulnerabilitiesForceDownloadProperty, repoPath);
```

Comment:
I've changed a bunch of log lines and exception messages so they're in the format of:

```
{what}. {why}. {where}.
```

A sort of hierarchy of uniqueness. Less unique on the left to more unique on the right. Makes logs look more consistent and easier to skim for details.